apps/quickstart-react/README.md

```diff
@@ -16,20 +16,20 @@ In the project directory, you should run:
 
 ### `npm install`
 
-And then to run an application with automatic virtual ngrok tunnel, run:
+And then to run an application with the monday tunnel, run:
 
 ### `npm start`
 
-Visit http://localhost:4040/status and under "command_line section" find the URL. This is the public URL of your app, so you can use it to test it.
-F.e.: https://021eb6330099.ngrok.io
+Find the provided URL in your terminal. This is your public URL, and you can use it to test your application.
+Example: https://unsightly-chickaree-35.tunnel.monday.app
```

Comment:
I think that URL with monday tunnel won't look this way? It composes the id of the application, so I would update example to look closer to what it be

Reply:
I know it's been 4 months, but...................

Actually, if you run it with npm run start (like in this instruction), the URL does look like this. 

It has the ID of the application only if you run it with the monday-cli scaffold command (which installs and starts the app in one shot).